templates/default/opensshd.conf.erb

```diff
@@ -193,6 +193,9 @@ UseDNS <%= ((@node['ssh-hardening']['ssh']['server']['use_dns']) ? 'yes' : 'no'
 #ChrootDirectory none
 #ChrootDirectory /home/%u
 
+#Accept locale environment variables
+AcceptEnv LANG LC_* LANGUAGE
```

Comment:
Could we have it configurable too? Attribute like `['ssh-hardening']['ssh']['server']['accept_env']` with `['LANG', 'LC_*', 'LANGUAGE']` as default in the `attributes/default.rb` would be cool